scripts/package-harvester-os

```diff
@@ -17,7 +17,7 @@ source ${SCRIPTS_DIR}/version-harvester ${TOP_DIR}/../harvester
 source ${SCRIPTS_DIR}/version-monitoring
 source ${SCRIPTS_DIR}/version-logging
 
-BASE_OS_IMAGE="rancher/harvester-os:20230615"
+BASE_OS_IMAGE="rancher/harvester-os:sle-micro-head"
```

Comment:
do we need to set a fixed version for a new release or plan to use the head instead?

Reply:
Yes, we need it. @Vicente-Cheng Please help create a new tag on the [harvester-v1.2.x](https://github.com/harvester/os2/tree/harvester-v1.2.x) branch.

Reply:
Hi @guangbochen,
We prefer to use sle-micro-head for the master branch.
And now, when we tag the OS image like `v1.2-20230629` or `v1.1-20230628` (for example).
The GitHub action will create PR to the related installer branch, so we could make **master** branch use **sle-micro-head** image for development. cc @bk201